File: PizzaBestilling/Models/Order.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace PizzaBestilling.Models
{
    public class Order
    {
        [Required]
        public int Id { get; set; }
        [Required]
        public string PizzaType { get; set; }
        [Required]
        public string PizzaStyle { get; set; }
        [Required]
        public int Quantity { get; set; }
        [Required]
        public string FullName { get; set; }
        [Required]
        public string Address { get; set; }
        [Required]
        public string Email { get; set; }
        [Required]
        public string PhoneNumber { get; set; }
    }
}

File: PizzaBestilling/ClientApp/src/types/Order.ts
export interface Order {
  id: number;
  pizzaType: string;
  pizzaStyle: string;
  quantity: number;
  fullName: string;
  address: string;
  email: string;
  phoneNumber: string;
}

export interface OrderForm {
  pizzaType: string;
  pizzaStyle: string;
  quantity: string;
  fullName: string;
  address: string;
  email: string;
  phoneNumber: string;
}
